app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,23 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
+        <activity android:name="lt.vilnius.tvarkau.StartActivity" />
+
+        <!-- This strangely named alias is here for backwards compatibility.
+             Renaming activity with LAUNCHER category means that shortcuts created by users
+             on their phones with previous version of the app will disappear or stop working. -->
+        <activity-alias
+            android:name="apps.factory.emiestas.MainActivity"
```

Comment:
Well, I'm working under assumption that we'll release the new app with the old package id. Otherwise we'll be left managing 2 apps instead of 1.